Makefile

```diff
@@ -100,18 +100,20 @@ bundle:
 	mkdir -p dist/$(PLUGIN_ID)
 	cp $(MANIFEST_FILE) dist/$(PLUGIN_ID)/
 ifneq ($(wildcard $(ASSETS_DIR)/.),)
-	cp -r $(ASSETS_DIR) dist/$(PLUGIN_ID)/
+	mkdir -p dist/$(PLUGIN_ID)/$(ASSETS_DIR)
+	cp -r $(ASSETS_DIR)/* dist/$(PLUGIN_ID)/$(ASSETS_DIR)/
```

Comment:
That change did broke CI as I doesn't work if`$ASSETS_DIR` is empty: https://app.circleci.com/pipelines/github/mattermost/mattermost-plugin-starter-template/340/workflows/87cc7120-9614-4702-9edf-b7b3607f05f3/jobs/717/parallel-runs/0/steps/0-103

Reply:
Mmm ok, yes I had thought the wildcard check above was already protecting against that case - evidently not.

Reply:
I suppose `ifneq ($(wildcard $(ASSETS_DIR)/.),)` actually says "if `ASSETS_DIR` is a directory".

Reply:
Then probably it's better to remove the `*` from all the recursive copies. I will test that locally (a bit more thoroughly)